cloud-gateway/src/main/resources/application.properties

```diff
@@ -1 +1,6 @@
-
+eureka.client.register-with-eureka = true
```

Comment:
Maybe you no longer need these eureka properties?

Reply:
Also, you seem to be using both .properties and .yaml files. Pick one and stick to it.